packages/zoe/src/zoeService/zoe.js

```diff
@@ -244,27 +244,23 @@ function makeZoe(vatAdminSvc, zcfBundleName = undefined) {
           getIssuers: () => instanceRecord.terms.issuers,
           getBrands: () => instanceRecord.terms.brands,
           getInstance: () => instance,
-          hasShutdown: () => hasShutdown,
-          shutdown: () => {
-            hasShutdown = true;
+          acceptingOffers: () => acceptingOffers,
+          exitAllSeats: () => {
+            acceptingOffers = false;
             zoeSeatAdmins.forEach(zoeSeatAdmin => zoeSeatAdmin.exit());
           },
+          kickOutAllSeats: reason => {
+            acceptingOffers = false;
+            zoeSeatAdmins.forEach(zoeSeatAdmin => zoeSeatAdmin.kickOut(reason));
+          },
+          stopAcceptingOffers: () => (acceptingOffers = false),
```

Comment:
We need to pipe this out to zcf to actually use this. I'm fine with creating an issue specifically for this, but I'm afraid that if we don't itemize it, we will forget and think it has happened when it hasn't